soh/soh/mixer.c

```diff
@@ -99,8 +99,14 @@ void aClearBufferImpl(uint16_t addr, int nbytes) {
     memset(BUF_U8(addr), 0, nbytes);
 }
 
-void aLoadBufferImpl(const void *source_addr, uint16_t dest_addr, uint16_t nbytes) {
+void aLoadBufferImpl(const void* source_addr, uint16_t dest_addr, uint16_t nbytes) {
+#if __SANITIZE_ADDRESS__
+    for (size_t i = 0; i < ROUND_DOWN_16(nbytes); i++) {
+        BUF_U8(dest_addr)[i] = ((const unsigned char*)source_addr)[i];
+    }
+#else
```

Comment:
i'm curious as to the reasoning behind doing this only when ASAN is enabled instead of always

Reply:
memcpy is guaranteed to never be slower than writing out the loop is the only reason.